src/core.c/Any-iterable-methods.pm6

```diff
@@ -1556,6 +1556,9 @@ Consider using a block if any of these are necessary for your mapping code."
         (find-reducer-for-op(&with))(&with,1)(self)
     }
 
+    proto method slice(|) is nodal { * }
+    multi method slice(Any:D: *@indices) { self.Seq.slice(@indices) }
```

Comment:
Why not provide a return Type for this?

Reply:
I guess one could.

Reply:
Added with https://github.com/rakudo/rakudo/commit/9e61a71be7